archeo/theme.json

```diff
@@ -256,6 +257,29 @@
 			"text": "var(--wp--preset--color--foreground)"
 		},
 		"elements": {
+			"button": {
+				"border": {
+					"radius": "0",
+					"width": "0"
+				},
+				"color": {
+					"background": "var(--wp--preset--color--foreground)",
+					"text": "var(--wp--preset--color--background)"
+				},
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--chivo)",
+					"fontSize": "var(--wp--preset--typography--font-size--normal)",
+					"textTransform": "uppercase"
+				},
```

Comment:
This doesn't seem to be getting picked up.  Is textTransform something that can be assigned in this way?

<img width="699" alt="image" src="https://user-images.githubusercontent.com/146530/177636429-5870428a-e60f-44bb-b177-8c1d2825788b.png">

Reply:
I think this was a bug with the elements API, and this has since been fixed. I've just had a play with moving it between `core/button` and the button element, and the button element seems to work consistently now 🤞

Reply:
Yup, works now. :)